bundle.md

```diff
@@ -10,7 +10,13 @@ A standard container bundle is made of the following 3 parts:
 - One or more content directories
 - A configuration file
 
-# Directory layout
+## Use Cases
+
+This bundle specification enables users in the following cases:
+
+- A user can create a root filesystem and configuration, with low-level OS and host specific details, and launch it as a container under an Open Container runner.
```

Comment:
runtime makes since to me too. I will fix that up here and then do a PR to make it consistent.